src/test/compile-fail/import-from-missing.rs

```diff
@@ -16,3 +16,4 @@ mod spam {
 }
 
 fn main() { ham(); eggs(); }
+//~^ ERROR unresolved name `eggs`
```

Comment:
hmm I'm trying to decide if I'll be annoyed by this message ("what are you talking about, I clearly tried to import it above!"), or if I will be happy that it is there.

I guess our user base *should* know enough to read and address the compiler error messages in order from top down...